docs/data/date-pickers/fields/LifeCycleDateRangeField.tsx

```diff
@@ -19,7 +19,7 @@ export default function LifeCycleDateRangeField() {
         <SingleInputDateRangeField
           value={value}
           onChange={(newValue) => setValue(newValue)}
-          sx={{ width: 400 }}
+          sx={{ width: 320 }}
```

Comment:
IMHO, in this specific case, having `300` is perfect. Or is there a case (size) where it's not enough? 🤔